packages/types/src/index.ts

```diff
@@ -217,7 +217,7 @@ export type Step = {
   area: boolean;
   bogusName: boolean;
   distance: number;
-  elevation: ElevationData;
+  elevationProfile: ElevationData[];
```

Comment:
To avoid breaking change and keep CI happy, let's keep the old field along with the new one.